index.html

```diff
@@ -91,37 +91,39 @@ <h3 class="blue-fg">LET'S CONNECT</h3>
+                        <div class="flex-box horizontal flex-center">
+                            <img class="portfolio_snapshot works-img-size" src="@images/Portfolio_1_snapshot.png" alt="Portfolio Snapshot image" />
+                        </div>
 
-                    <!-- Portfolio description (FLEX vertical) -->
-                    <div >
-                        <h2></h2>
-                        <div>
-                            <span class=""></span>
-                            <span class=""></span>
-                            <span class=""></span>
-                            <span class=""></span>
-                            <span class=""></span>
+                        <!-- Portfolio description (FLEX vertical) -->
+                        <div class="flex-box vertical flex-space-between">
+                            <h2 class="work-title-props">Tonic</h2>
+                            <div class="flex-box horizontal flex-start work-card-h-gap-12px">
+                                <span class="client-name-props">CANOPY</span>
+                                <img class="bullet-margin" src="@images/small_bullet.svg" alt="Bullet">
+                                <span class="work-name-props">Back End Dev</span>
+                                <img class="bullet-margin" src="@images/small_bullet.svg" alt="Bullet">
+                                <span class="work-year-props">2015</span>
+                            </div>
+                            <p class="black-fg word-card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
                         </div>
-                        <p></p>
-                    </div>
 
-                    <!-- Programming Languages Used (FLEX vertical) -->
-                    <div >
-                        <ul>
-                            <li></li>
-                            <li></li>
-                            <li></li>
-                        </ul>
-                    </div>
+                        <!-- Programming Languages Used (FLEX vertical) -->
+                        <div class="flex-box horizontal flex-start">
+                            <ul class="no-bullet flex-box horizontal flex-start no-padding work-card-h-gap-8px">
+                                <li class="prog-lang-list r-margin-8px">html</li>
+                                <li class="prog-lang-list r-margin-8px">css</li>
+                                <li class="prog-lang-list">javaScript</li>
+                            </ul>
+                        </div>
 
-                    <!-- See Project button (FLEX vertical) -->
-                    <div >
-                        <button></button>
+                        <!-- See Project button (FLEX vertical) -->
+                        <div class="flex-box horizontal flex-start">
+                            <button class="works-card-button">See Project</button>
```

Comment:
- [ ] Good job designing your project cards 💯 but your `See project` button has not been designed well. kindly address this issue so as to follow the template.
- [ ] Also you'll notice that the background color of the page is different from the background color of the project cards. It would be nice to fix this so as to improve the user interface 😄 
- Your design:
![Annotation 2022-02-20 070044](https://user-images.githubusercontent.com/85113567/154831050-07e09b42-9283-44b3-99f0-c2c78316df87.jpg)
- Figma design:
![Annotation 2022-02-20 070104](https://user-images.githubusercontent.com/85113567/154831054-330660a9-5a51-4ab7-a159-61d9d74caa2f.jpg)